charts/incubator/sonsoftheforest/templates/_configmap.tpl

```diff
@@ -0,0 +1,55 @@
+{{/* Define the configmap */}}
+{{- define "sonsoftheforest.configmaps" -}}
+{{- $fullname := (include "tc.v1.common.lib.chart.names.fullname" $) -}}
+
+{{- $server := .Values.sonsoftheforest.server -}}
+
+sonsoftheforest-dscfg:
+  enabled: true
+  data:
+    dedicatedserver.cfg: |
+      {
+        "IpAddress": "0.0.0.0",
+        "GamePort": "{{ .Values.service.main.ports.main.port }}",
+        "QueryPort": "{{ .Values.service.query.ports.query.port }}",
+        "BlobSyncPort": "{{ .Values.service.sync.ports.sync.port }}",
+        "ServerName": "{{ $server.ServerName }}",
+        "MaxPlayers": "{{ $server.MaxPlayers }}",
+        "Password": "{{ $server.Password }}",
+        "LanOnly": "{{ $server.LanOnly }}",
+        "SaveSlot": "{{ $server.SaveSlot }}",
+        "SaveMode": "{{ $server.SaveMode }}",
+        "GameMode": "{{ $server.GameMode }}",
```

Comment:
why not do all this like `eco` with `toJson`?

Reply:
i wasnt sure how "GameSettings" and "CustomGameModeSettings" would translate over correctly or not.